pkgs/development/libraries/bamf/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, fetchurl, libgtop, libwnck3, glib, vala, pkgconfig
+, libstartup_notification, gobjectIntrospection, gtk-doc
+, python27, pythonPackages, libxml2 }:
+
+stdenv.mkDerivation rec {
+  pname = "bamf";
+  version = "0.5.3";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://launchpad.net/${pname}/0.5/${version}/+download/${name}.tar.gz";
+    sha256 = "051vib8ndp09ph5bfwkgmzda94varzjafwxf6lqx7z1s8rd7n39l";
+  };
+
+  propagatedBuildInputs = [glib gobjectIntrospection];
```

Comment:
Why is it propagating here?

Reply:
i just copied it from some other package. changed